contracts/src/Zappers/GasCompZapper.sol

```diff
@@ -191,6 +191,8 @@ contract GasCompZapper is AddRemoveManagers, LeftoversSweep {
         bool _isDebtIncrease,
         InitialBalances memory _initialBalances
     ) internal returns (address) {
+        require(!_isDebtIncrease || _boldChange > 0, "GCZ: Increase bold amount should not be zero");
+
```

Comment:
Now it's consistent with WETHZapper, but it still seems inconsistent between coll & debt to me (this goes for WETHZapper, too):

- `isDebtIncrease = true` requires `_boldChange > 0`
- `isCollIncrease = true` doesn't require `_collChange > 0`, in fact you _must_ pass `_isCollIncrease = true` along with `_collChange = 0` if you are add-manager but not remove-manager and want to repay some debt but not add any collateral.

Wouldn't this be better?
```solidity
if ((!_isCollIncrease && _collChange > 0) || (_isDebtIncrease && _boldChange > 0)) {
    receiver = _requireSenderIsOwnerOrRemoveManagerAndGetReceiver(_troveId, owner);
}

if ((_isCollIncrease && _collChange > 0) || (!_isDebtIncrease && _boldChange > 0)) {
    _requireSenderIsOwnerOrAddManager(_troveId, owner);
}
```

Then we could drop the `require(!_isDebtIncrease || _boldChange > 0)`.

Reply:
For now I put those conditions in a common `BaseZapper` contract... At least we ensure consistency :sweat_smile:

Reply:
Yeah, that looks good!